<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="1" skipped="0" tests="2" time="109.486" timestamp="2022-12-12T11:31:24.529523" hostname="C02Z85AULVDC"><testcase classname="tests.test_components.test_mcms" name="test_download_mouse" time="37.452" /><testcase classname="tests.test_components.test_mcms" name="test_get_mouse_df" time="71.748"><failure message="OSError: Cannot find downloaded file starting with PZAJ2.1c_get_mouse_df_file in this download folder:&#10;/Users/blota/Downloads">@pytest.mark.slow
    def test_get_mouse_df():
&gt;       mcms.get_mouse_df(mouse_name="PZAJ2.1c", username=USERNAME)

tests/test_components/test_mcms.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

mouse_name = 'PZAJ2.1c', username = 'ab8', password = None

    def get_mouse_df(mouse_name, username, password=None):
        """Load mouse info from mcms in a dataframe"""
        ret = download_mouse_info(
            mouse_name, username, password, suffix="get_mouse_df_file"
        )
        if not ret:
            raise IOError("Failed to download mouse info")
        fnames = [
            s
            for s in os.listdir(PARAMETERS["download_folder"])
            if s.startswith(mouse_name + "_get_mouse_df_file")
        ]
        if len(fnames) &gt; 1:
            raise IOError(
                "Multiple file found. Please remove old downloads with similar name "
                "(i.e. starting with %s" % (mouse_name + "_get_mouse_df_file")
            )
        if not len(fnames):
&gt;           raise IOError(
                "Cannot find downloaded file starting with %s in this download folder:\n%s"
                % (mouse_name + "_get_mouse_df_file", PARAMETERS["download_folder"])
            )
E           OSError: Cannot find downloaded file starting with PZAJ2.1c_get_mouse_df_file in this download folder:
E           /Users/blota/Downloads

flexiznam/mcms/main.py:82: OSError</failure></testcase></testsuite></testsuites>